data/remnant/remnant events.txt

```diff
@@ -39,12 +39,13 @@ event "remnant: defense timer"
 event "remnant: surveillance begin"
 	system "Parca"
 		remove fleet "Korath Ember Waste Raid"
-
+		remove fleet "Kor Alaji Raid"
 
 
 event "remnant: surveillance end"
 	system "Parca"
-		add fleet "Korath Ember Waste Raid" 10000
+		add fleet "Kor Alaji Raid" 20000
+		add fleet "Korath Ember Waste Raid" 15000
```

Comment:
Why did the spawn rates change from before the event?

Reply:
Good catch.

```suggestion
		add fleet "Kor Alaji Raid" 30000
		add fleet "Korath Ember Waste Raid" 30000
		add fleet "Kor Isaha Raid" 15000
```